server/config.js

```diff
@@ -107,6 +107,26 @@ var conf = convict({
     env: 'REDIS_DATABASE',
     format: 'int'
   },
+  db_reindex_timeout: {
+    doc: 'Number of milliseconds for etl to fetch from postgres + bulk insert into elasticsearch',
```

Comment:
ah, whoops. I initially called this 'etl' code, as in extract-transform-load. I can remove those refs from the config file.